dev/docker/hive/download-release.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# gh-dl-release! It works!
+# 
+# This script downloads an asset from latest or specific Github release of a
+# private repo. Feel free to extract more of the variables into command line
+# parameters.
+#
+# PREREQUISITES
+#
+# curl, wget, jq
+#
+# USAGE
+#
+# Set all the variables inside the script, make sure you chmod +x it, then
+# to download specific version to my_app.tar.gz:
+#
+#     gh-dl-release 2.1.1 my_app.tar.gz
```

Comment:
- how about make the syntax something like `<script> <package> [release]` with `release` defaults to `latest`?
- is the `.tar.gz` necessary?

I'm thinking something like this:

```
download-release.sh ranger
download-release.sh ranger 2.4.0
```